src/components/heroCanvas.js

```diff
@@ -91,17 +91,18 @@ const drawHeroBg = (canvas, imageEl) => {
 
 const HeroCanvas = () => {
   const canvasContainerRef = useRef(null)
-  const canvasRef = useRef(null)
   const [imageEl, setImageEl] = useState(null)
 
   const initializeCanvas = () => {
     const { clientWidth, clientHeight } = canvasContainerRef.current
-    const canvas = new fabric.Canvas(canvasRef.current, {
+    const canvas = document.createElement("canvas")
+    canvasContainerRef.current.innerHTML = ""
+    canvasContainerRef.current.appendChild(canvas)
```

Comment:
Append to and clear the canvas container imperatively since `new fabric.Canvas()` adds elements beyond just the HTML Canvas, which need to be cleared on initialization after resizing.